pallets/common/src/protocol_fee.rs

```diff
@@ -0,0 +1,34 @@
+use codec::{Decode, Encode};
+use frame_support::dispatch::DispatchResult;
+use polymesh_primitives::Signatory;
+#[cfg(feature = "std")]
+use sp_runtime::{Deserialize, Serialize};
+
+/// Protocol fee operations.
+#[derive(Decode, Encode, Clone, Copy, Debug, Hash, PartialEq, Eq, PartialOrd, Ord)]
+#[cfg_attr(feature = "std", derive(Serialize, Deserialize))]
+pub enum ProtocolOp {
```

Comment:
Shouldn't this file come under the `traits` folder structure?

Reply:
I don't think `ProtocolOp` should be in `traits`. As for `ChargeProtocolFee`, we might but in that case we'd just have another tiny module under `traits` and I'd export it from the root of the crate to save on the associated refactoring. What is you final word, @satyamakgec ?

Reply:
I left it as is for now.

Reply:
Sorry for my unclear comment, I was saying that whole `protocol_fee` should be a part of `traits` folder